### PR TITLE
File Setup

### PR DESCRIPTION
**Pull request by Joe-Bentley**
#1 See Slack for admin login details

### CODE REVIEW

File: totems/settings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
```
Comment (nmunro) [inline totems/settings.py:33]: Remember to install your app here

**Review by nmunro — DISMISSED** [review]

File: totems/settings.py (hunk 1 above)
Comment (maddyharrop) [reply totems/settings.py:33]: totems.apps.TotemsConfig ?

Comment (maddyharrop) [reply totems/settings.py:33]: or artifacts.apps.AtifactsConfig ?

**Review by maddyharrop — APPROVED** [review]
Very good start

**Pull request merged**